migrations/20240228-add_column_templateVersionId_to_pipeline.JS

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}pipelines`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async transaction => {
+            await queryInterface.addColumn(
+                table,
+                'templateVersionId',
+                {
+                    type: Sequelize.INTEGER
+                },
+                { transaction }
+            );
+
+            await queryInterface.addIndex(table, ['templateVersionId'], {
+                name: `${table}_template_version_id`,
+                transaction
+            });
```

Comment:
Per doc, it is suggested to use `options.fields`
https://sequelize.org/api/v6/class/src/dialects/abstract/query-interface.js~queryinterface#instance-method-addIndex
```suggestion
            await queryInterface.addIndex(table, {
                name: `${table}_template_version_id`,
                fields: ['templateVersionId'],
                transaction
            });
```